packages/search/src/types.ts

```diff
@@ -29,3 +31,22 @@ export class FileError extends Error {
 }
 
 export type ExtractType = "styleFiles" | "componentFiles" | "themeFiles";
+
+export interface Declarations {
```

Comment:
If this is just a mapping between declaration name and source, would it make more sense to rename the type to `DeclarationSources`?